spec/features/pets/pets_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe "pets index page", type: :feature do
+  it "can list all pets on pets index page" do
+    shelter_1 = Shelter.create!(name: "Denver Animal Shelter",
+                               address: "500 Invisible St.",
+                               city: "Denver",
+                               state: "Colorado",
+                               zip: "80201")
+
+    pet = Pet.create!(image: 'https://image.shutterstock.com/z/stock-photo-attentive-border-collie-dog-lying-down-on-the-grass-on-a-sunny-d-593634296.jpg',
+                     name: 'Rover',
+                     age: 3,
+                     sex: "Male",
```

Comment:
Tests could be more robust throughout all tests, I am more comfortable with capybara then when I started this project so will go back and update tests to be more robust